person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: false)
+    @id = rand(999)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services
+    @age >= 18 || @parent_permission
+  end
```

Comment:
- [ ] Since we instead returning `true` or `false` from the `can_use_services` method, I think it would be more expressive to add the question mark (?) so it becomes `can_use_services?`

- [ ] We don't need to repeat `@age >= 18` in the `can_use_services?` method. We already have a method `of_age?` that checks for that so we should use that instead.

- [ ] **[OPTIONAL]** Since all methods are public by default, we can arrange our code so that all public methods are above the `private` keyword instead of using `public` this way.

  ```
  ...
  
  def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age?
    @age >=18
  end
  ```